pkg/util/namespace/namespace.go

```diff
@@ -23,3 +23,69 @@ func IsOpenShiftSystemNamespace(ns string) bool {
+		"openshift-cluster-samples-operator":               true,
+		"openshift-cluster-storage-operator":               true,
+		"openshift-config":                                 true,
+		"openshift-config-managed":                         true,
+		"openshift-config-operator":                        true,
+		"openshift-console":                                true,
+		"openshift-console-operator":                       true,
+		"openshift-console-user-settings":                  true,
+		"openshift-controller-manager":                     true,
+		"openshift-controller-manager-operator":            true,
+		"openshift-dns":                                    true,
+		"openshift-dns-operator":                           true,
+		"openshift-etcd":                                   true,
+		"openshift-etcd-operator":                          true,
+		"openshift-host-network":                           true,
+		"openshift-image-registry":                         true,
+		"openshift-ingress":                                true,
+		"openshift-ingress-canary":                         true,
+		"openshift-ingress-operator":                       true,
+		"openshift-insights":                               true,
+		"openshift-kni-infra":                              true,
+		"openshift-kube-apiserver":                         true,
+		"openshift-kube-apiserver-operator":                true,
+		"openshift-kube-controller-manager":                true,
+		"openshift-kube-controller-manager-operator":       true,
+		"openshift-kube-scheduler":                         true,
+		"openshift-kube-scheduler-operator":                true,
+		"openshift-kube-storage-version-migrator":          true,
+		"openshift-kube-storage-version-migrator-operator": true,
+		"openshift-machine-api":                            true,
+		"openshift-machine-config-operator":                true,
+		"openshift-marketplace":                            true,
+		"openshift-monitoring":                             true,
+		"openshift-multus":                                 true,
+		"openshift-network-diagnostics":                    true,
+		"openshift-network-operator":                       true,
+		"openshift-oauth-apiserver":                        true,
+		"openshift-openstack-infra":                        true,
+		"openshift-operator-lifecycle-manager":             true,
+		"openshift-operators":                              true,
+		"openshift-ovirt-infra":                            true,
+		"openshift-sdn":                                    true,
+		"openshift-service-ca":                             true,
+		"openshift-service-ca-operator":                    true,
+		"openshift-user-workload-monitoring":               true,
+		"openshift-vsphere-infra":                          true,
+		"openshift-azure-operator":                         true,
+		"openshift-managed-upgrade-operator":               true,
+	}
+	return filtered_namespaces[ns]
```

Comment:
We should use a `map[string]struct{}` here instead of `map[string]bool` as golang has a construct when accessing values out of a map to determine if they exist.  We use this pattern in a few places. 

https://github.com/Azure/ARO-RP/blob/master/cmd/aro/mirror.go#L25-L28
https://github.com/Azure/ARO-RP/blob/master/cmd/aro/mirror.go#L112-L114

you can then do something like

``` 
_, ok := filtered_namespaces[ns]
return ok
```

Reply:
Thanks Ben, I have updated the code based on your suggestions in the new commit.